### DISCUSSION TITLE
Erro atualizar mapos 4.42.0

### DISCUSSION THREAD

**Discussion by OsvaldoTecnovel** [RamonSilva20/mapos#2337] [General]
Como corrigir esse erro ao atulizar mapos? Estou tentando atualizar logando no sistema.
![Erro_atualizar mapos 4 42 0](https://github.com/RamonSilva20/mapos/assets/145617362/a09bbdf3-6e41-4042-ad7d-a9c95beb988c)

**Comment by Fesantt** [comment]
o updater automatico esta apresentando problemas no momento, só sera possivel a Atualização manual

**Comment by lucasbmira1** [comment]
![image](https://github.com/user-attachments/assets/312b2c0a-8ef3-4518-b1b4-6e3e3f1aeefc)
O meu deu um erro diferente.. Não consigo atualizar, nem instalar direto a nova versão online.. 

Estou travado na 4.40 sem dashboard de recebimentos
